tests/test_rtc_gamma.py

```diff
@@ -1,18 +1,28 @@
 import json
-import subprocess
 from glob import glob
 from pathlib import Path
 from time import sleep
 
 import pytest
+import xarray as xr
 
 from hyp3_testing import API_TEST_URL, API_URL
+from hyp3_testing import compare
 from hyp3_testing import helpers
 
 pytestmark = pytest.mark.golden
 _API = {'main': API_URL, 'develop': API_TEST_URL}
 
 
+def _get_tolerances(file_name):
```

Comment:
Should "tif" be in the name here (or maybe the input variable name)?

Reply:
renamed to `_get_tif_tolerances`